src/server/plugins/register-form/index.js

```diff
@@ -42,5 +46,5 @@ exports.register = function(server, options, next) {
 exports.register.attributes = {
   name: 'RegistrationFormRoutes',
   version: '1.0.0',
-  dependencies: 'NunjucksConfig'
+  dependencies: ['NunjucksConfig','crumb']
```

Comment:
And trailing `,`? What's our lint setup for that? Seems like we're not enforcing trailing commas actually...